lookout/style/typos/preprocessing.py

```diff
@@ -0,0 +1,35 @@
+from typing import Set, Union
+
+import pandas
+
+from lookout.style.typos.utils import Columns
+
+
+def check_split(split: str, tokens_set: Set[str]) -> bool:
+    """
+    Check whether all elements of the split are in tokens_set.
+
+    :param split: String of space-separated tokens.
+    :param tokens_set: Set of tokens (reference vocabulary).
+    :return: True when all tokens of split belong to tokens_set.
+    """
+    if type(split) != str:
+        return False
+
+    for token in split.split():
+        if token not in tokens_set:
+            return False
+    return True
+
+
+def filter_splits(data: Union[pandas.DataFrame, str], tokens_set: Set[str]) -> pandas.DataFrame:
```

Comment:
`token_set` -> `token`

And data type is just `pandas.DataFrame` as I can see.